README.md

```diff
@@ -33,7 +33,17 @@ Anna is part of KWOC : You can talk to us on Zulip or comment on issues to start
   - If I don't understand
   - Close tabs
   
-
+ # How to install ANNA
+ 
+1. Once you are on the repository page,look for the green “Clone or download” button on the right-hand side of the page.
+https://user-images.githubusercontent.com/29461633/33721594-8bb28f76-db8d-11e7-8e18-6d53aef7e9a2.png
+2. Click the “Clone or download” button and then click “Download ZIP.”
+https://user-images.githubusercontent.com/29461633/33721691-bb2e32a0-db8d-11e7-9452-a73b1b6ec4a8.png
+3. Take the downloaded ZIP file and extract it. We need to give it a permanent home because Chrome will externally load the files (if they get moved/deleted then the extension will stop working).
+4. With the extension downloaded and extracted, we can now install it.In Chrome, navigate to chrome://extensions
+5. Installing the extension is extremely easy. All we need to do is drag-and-drop the folder we extracted onto Chrome’s Extensions page.If the extension is in the “.crx” format, Chrome will ask you to review and approve the permissions before installing. 
```

Comment:
modify this part a but and add image for this

Reply:
point 5.